src/viam/app/app_client.py

```diff
@@ -1581,25 +1632,25 @@ async def delete_robot(self, robot_id: str) -> None:
         request = DeleteRobotRequest(id=robot_id)
         await self._app_client.DeleteRobot(request, metadata=self._metadata)
 
-    async def list_fragments(self, org_id: str, show_public: bool = True) -> List[Fragment]:
+    async def list_fragments(self, org_id: str, show_public: bool = True, visibilities: List[FragmentVisibility] = []) -> List[Fragment]:
         """Get a list of fragments under the currently authed-to organization.
 
         ::
 
-            fragments_list = await cloud.list_fragments(org_id="org-id", show_public=False)
+            fragments_list = await cloud.list_fragments(org_id="org-id", visibilities=[])
 
         Args:
             org_id (str): The ID of the organization to list fragments for.
                 You can obtain your organization ID from the Viam app's organization settings page.
-            show_public: Optional boolean specifying whether or not to only show public fragments. If True, only public fragments will
-                return. If False, only private fragments will return. Defaults to True.
+            visibilities: List of FragmentVisibilities specifying which types of fragments to include in the results.
+                If empty, by default only public fragments will be returned.
 
         Returns:
             List[viam.app.app_client.Fragment]: The list of fragments.
 
         For more information, see `Fleet Management API <https://docs.viam.com/appendix/apis/fleet/>`_.
         """
-        request = ListFragmentsRequest(organization_id=org_id, show_public=show_public)
+        request = ListFragmentsRequest(organization_id=org_id, fragment_visibility=map(Fragment.visibility_to_proto, visibilities))
```

Comment:
Similarly, I think you should continue to include `show_public` since we haven't pulled it out yet, giving people time to migrate to the new visibilities workflow

Reply:
On the app side, we have completely removed any references to show_public, so passing it here wouldn't do anything. I can add a comment stating it is deprecated though

Reply:
Nevermind! Talked with Katherine and we will be adding the `show_public` functionality on the app side

Reply:
Yeah , spoke with Michael this morning and he suggested we just accommodate both fields again for a while